tests/test_bootstrap.py

```diff
@@ -1 +1,106 @@
-from straPy import bootstrap
+import numpy as np
+from pytest import raises
+from straPy.bootstrap import bootstrap_distribution
+
+
+def test_bootstrap_distribution():
+    """
+    Tests the functionality of `bootstrap_distribution()`, which includes
+    checking the type of object returned, it's shape and it's values.
+
+    5 tests in total.
+    """
+
+    dist = bootstrap_distribution([1, 2, 3], 3, 3, random_seed=1)
+    
+    # checks the return type
+    assert isinstance(dist, np.ndarray)
+
+    # checks the shape of returned object
+    assert dist.shape == (3,)
+
+    # checks the values of returned object
+    assert np.array_equal(dist.tolist(),
+                          [1.3333333333333333,
+                           1.6666666666666667,
+                           1.3333333333333333])
+
+    dist = bootstrap_distribution([1, 2, 3],
```

Comment:
```suggestion
    dist_var_1 = bootstrap_distribution([1, 2, 3],
```

Reply:
I got confused in the later test because dist was reassigned, renaming for clarity